src/layouts/FeedbackDialog.tsx

```diff
@@ -0,0 +1,109 @@
+import { Trans } from '@lingui/macro';
+import { Box, Button, CircularProgress, TextField, Typography } from '@mui/material';
+import { FormEvent, useEffect, useState } from 'react';
+import { BasicModal } from 'src/components/primitives/BasicModal';
+import { BaseSuccessView } from 'src/components/transactions/FlowCommons/BaseSuccess';
+import { useRootStore } from 'src/store/root';
+
+export const FeedbackModal = () => {
+  const [feedbackDialogOpen, setFeedbackOpen] = useRootStore((state) => [
+    state.feedbackDialogOpen,
+    state.setFeedbackOpen,
+  ]);
+
+  const [value, setValue] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [success, setSuccess] = useState(false);
+
+  useEffect(() => {
+    if (feedbackDialogOpen) {
+      setSuccess(false);
+    }
+  }, [feedbackDialogOpen]);
+
+  const handleFeedbackSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!value) {
+      return alert('Please add feedback');
+    }
+
+    setIsLoading(true);
+
+    const url = process.env.NEXT_PUBLIC_API_BASEURL + '/feedback';
+
+    const payload = {
+      text: value,
+    };
+
+    try {
+      await fetch(url, {
+        body: JSON.stringify(payload),
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+
+      setIsLoading(false);
+      setSuccess(true);
+      setValue('');
+    } catch (error) {
+      setIsLoading(false);
+      setValue('');
+
+      return alert('Submission did not work, please try again later or contact wecare@avara.xyz');
```

Comment:
Instead of an alert, I think it'd be better to add an error message to the modal